docs/specs/restore.yml

```diff
@@ -204,11 +204,7 @@ repos:
 cache:
   .lock.json: |
     {
-      "git":{
-         "https://docs.com/lock1/c#master":{
-            "commit":"8720d784fb99abcbdfa922025b8bcc060f0f622a"
-         }
-      }
+      "https://docs.com/lock1/c#master":"8720d784fb99abcbdfa922025b8bcc060f0f622a"
```

Comment:
Use array of objects in persistent data model, this give us ways to upgrade in the future without breaking changes. Most of existing persistent data model follows this principle:

{
   "git": [
       { "url": "", "branch": "", "commit": "" },
       { "url": "", "branch": "", "commit": "" },
    ]
}

Reply:
array would make it hard for lookup. how about use remote url as key